### PR TITLE
Add a notification to update skeleton dashboards

### PR DESCRIPTION
**Pull request by fejta**
/assign @michelle192837 @grodrigues3 @rmmh

### CODE REVIEW

**Comment by grodrigues3** [comment]
What are notifications?  Where do they show up and what information is supposed to go in them?  Might be worth updating the docs here too:

https://github.com/kubernetes/test-infra/tree/master/testgrid/config#add-a-notification

**Comment by fejta** [comment]
Done

File: testgrid/config/README.md [hunk 1]
```diff
@@ -113,35 +113,48 @@ Narrow down where to search when searching for a regression between two builds/c
...
-test_groups:
-- name: kubernetes-build
-  gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-build
-  notifications:
-  - summary: I am a test group notification.
-  - summary: Tests are failing to start.
-    context_link: https://github.com/kubernetes/kubernetes/issues/123
-```
+This is an effective way to broadcast system wide information (all
+FOO suites are failing due to blah, upgrade frobber to vX before the
```
Comment (grodrigues3) [inline testgrid/config/README.md]: is that supposed to say frobber?

Comment (fejta) [inline testgrid/config/README.md]: Yes

**Comment by fejta** [comment]
I'll update the README.md in a separate PR as there are other changes I want to make

**Comment by fejta** [comment]
PTAL

**Pull request merged**